Backend/src/main/java/com/onbank/api/model/Transfer.java

```diff
@@ -14,11 +17,8 @@
 @Data
```

Comment:
zmien na getter and setter jezeli korzystamy osobno z adnotacji equals and hash code

Reply:
Zrobione